src/main/scala/de/frosner/dds/core/DDS.scala

```diff
@@ -642,4 +648,19 @@ object DDS {
     summarizeGroups(toBeGroupedValues.groupByKey())
   }
 
+  private[core] def lg2IntCeil(count:Long) : Integer = {
```

Comment:
Does it make sense to put this into a Math util class somewhere? DDS.scala is becoming quite full. If so, can you create an issue to do it and give it the newbie label?